DataFormats/PortableMath/interface/deltaPhi.h

```diff
@@ -0,0 +1,45 @@
+#ifndef DataFormats_PortableMath_deltaPhi_h
+#define DataFormats_PortableMath_deltaPhi_h
+
+/* functions to compute heterogeneous deltaPhi
+ * Ported from the LST algorithm:
+ * https://github.com/cms-sw/cmssw/pull/45117#discussion_r1749143439
```

Comment:
IIRC, `reduceRange` is taken from DataFormats/Math/interface/deltaPhi.h
So, LST is mis-attribution.